lib/inferno_requirements_tools/tasks/collect_requirements.rb

```diff
@@ -0,0 +1,228 @@
+# frozen_string_literal: true
+
+require 'CSV'
+require 'roo'
+require 'yaml'
+require 'csv'
+
+module InfernoRequirementsTools
+  module Tasks
+    # This class manages the collection of requirements details from
+    # requirements planning excel workbooks into a CSV representation.
+    # Currently splits out Requirements and Planned Not Tested Requirements
+    # into two separate files.
+    #
+    # The `run_check` method will check whether the previously generated file is up-to-date.
+    class CollectRequirements
+      CONFIG = YAML.load_file(File.join('lib', 'requirements_config.yaml'))
+
+      TEST_KIT_ID = CONFIG['test_kit_id']
+      INPUT_SETS = CONFIG['requirement_sets'].map do |req_set|
+        req_set['id']
+      end
+
+      # Derivative constants
+      TEST_KIT_CODE_FOLDER = TEST_KIT_ID.gsub('-', '_')
+      DASHERIZED_TEST_KIT_ID = TEST_KIT_ID.gsub('_', '-')
```

Comment:
If you want the naming of the generated requirements files to have underscores rather than dashes, this DASHERIZED_TEST_KIT_ID needs to be removed in this collect_requirements file

Reply:
moving requirements conversion to another branch - hoping to merge this one first